tests/console/command_not_found.pm

```diff
@@ -27,9 +28,25 @@ sub run {
     }
 
     my $not_installed_pkg = (is_sle && sle_version_at_least '15') ? 'wireshark' : 'xosview';
-    assert_script_run("echo \"\$(cnf $not_installed_pkg 2>&1 | tee /dev/stderr)\" | grep -q \"zypper install $not_installed_pkg\"");
+    my $cnf_cmd           = "echo \"\$(cnf $not_installed_pkg 2>&1 | tee /dev/stderr)\" | grep -q \"zypper install $not_installed_pkg\"";
+    my $assert            = 1;
+    if (sle_version_at_least '15') {
+        my $assert = script_run($cnf_cmd);
+        select_console 'root-console';
+        record_soft_failure 'https://fate.suse.com/323424';
+        add_suseconnect_product("sle-module-desktop-applications");
+        select_console 'user-console';
+    }
+    assert_script_run $cnf_cmd if $assert;    # Run command if not yet executed (workaround for SLE 15)
     save_screenshot;
 }
 
+# deativate desktop applications module on sle 15
+sub post_run_hook {
+    select_console 'root-console';
```

Comment:
Herr I would only execute the code if the above registration was actually executed